### PR TITLE
The Witness: Update docs (outdated information)

### PR DESCRIPTION
**Pull request by NewSoupVi**

### CODE REVIEW

File: worlds/witness/docs/en_The Witness.md [hunk 1]
```diff
@@ -10,12 +10,13 @@ config file.
 Puzzles are randomly generated using the popular [Sigma Rando](https://github.com/sigma144/witness-randomizer).
 They are made to be similar to the original game, but with different solutions.
 
-Ontop of that each puzzle symbol (Squares, Stars, Dots, etc.) is now an item.
+Ontop of that, each puzzle symbol (Squares, Stars, Dots, etc.) is now an item.
```
Comment (LegendaryLinux) [inline worlds/witness/docs/en_The Witness.md:13]: "Ontop" should be `On top`.

File: worlds/witness/docs/en_The Witness.md [hunk 2]
```diff
@@ -10,12 +10,13 @@ config file.
...
 Panels with puzzle symbols on them are now locked initially.
 
 ## What is a "check" in The Witness?
 
 Solving the last panel in a row of panels or an important standalone panel will count as a check, and send out an item.
+It is also possible to add Environmental Puzzles into the location pool via a setting.
```
Comment (LegendaryLinux) [inline worlds/witness/docs/en_The Witness.md:19]: It may be a good idea to give the name of the setting here.

**Review by LegendaryLinux — CHANGES_REQUESTED** [review]

File: worlds/witness/docs/en_The Witness.md (hunk 2 above)
Comment (NewSoupVi) [reply worlds/witness/docs/en_The Witness.md:19]: The website display name or the yaml name?

Comment (LegendaryLinux) [reply worlds/witness/docs/en_The Witness.md]: I'd use the website display name, as modifying a `yaml` file directly is considered "advanced" usage.

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**